Dockerfile

```diff
@@ -6,7 +6,7 @@
 ARG BASE_IMG=nvcr.io/nvidia/cuda
 ARG BASE_IMG_TAG=12.5.1-base-ubuntu22.04
 
-# Use NVIDIA Morpheus as the base image
+# Use NVIDIA cuda as the base image
```

Comment:
Just to note: we should probably change this at some point  too, if we build from a stripped down ubuntu22.04 we'll probably be a lot lighter than with the cuda container.

Reply:
Big fan of this. Will do in a follow up PR.